waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/dto/BriefAssetDescription.scala

```diff
@@ -2,15 +2,16 @@ package com.wavesplatform.dex.grpc.integration.dto
 
 import com.wavesplatform.transaction.assets.exchange.AssetPair
 
-case class BriefAssetDescription(name: String, decimals: Int, hasScript: Boolean)
+case class BriefAssetDescription(name: String, decimals: Int, hasScript: Boolean, isNft: Boolean = false)
```

Comment:
A new field with a default value is bad for structures, because:
* The default value it is not always appropriate;
* There could be forgotten cases, thus the compiler won't force us to think about each case.